content/zh/docs/hertz/tutorials/service-governance/service_discovery.md

```diff
@@ -16,6 +16,57 @@ description: >
 - [polaris](https://github.com/hertz-contrib/registry/tree/main/polaris)
 - [servicecomb](https://github.com/hertz-contrib/registry/tree/main/servicecomb)
 - [zookeeper](https://github.com/hertz-contrib/registry/tree/main/zookeeper)
+- [redis](https://github.com/hertz-contrib/registry/tree/main/redis)
+
+## 配置
+
+使用服务发现时会提供一些可选配置给用户。
+
+| 配置                   | 描述                                       |
+| ---------------------- | ------------------------------------------ |
+| WithSD                 | 设置 isSD 为 true ，**使用服务发现必选**。 |
+| WithTag                | 为 requestOptions 设置 tag 。              |
+| WithCustomizedAddrs    | 自定义目标实例地址。                       |
+| WithLoadBalanceOptions | 配置负载均衡选项。                         |
+
+### WithSD
+
+`requestOptins` 提供 `WithSD` 配置项，传入参数为 true 时将会设置 `isSD` 为 true。使用服务发现请求时必须使用 `WithSD` 配置项。
+
+示例代码：
+```go
+status, body, err := cli.Get(context.Background(), nil, "http://hertz.test.demo/ping", config.WithSD(true))
+```
+
+### WithTag
+
+`requestOptins` 提供 `WithTag` 配置项，使用此配置会为 `requestOptions` 设置 tag。
+
+示例代码：
+```go
+status, body, err := cli.Get(context.Background(), nil, "http://hertz.test.demo/ping", config.WithTag("foo", "var"))
+```
+
+### WithCustomizedAddrs
+
+此配置项指定服务发现时的目标实例地址。它将会覆盖来自 `Resolver` 的结果。
```

Comment:
Maybe we could make a brief introduction about `Resolver`.

Reply:
done, thx.